video/out/vo.c

```diff
@@ -70,8 +70,8 @@ extern const struct vo_driver video_out_kitty;
 static const struct vo_driver *const video_out_drivers[] =
 {
     // high-quality and well-supported VOs first:
-    &video_out_gpu,
     &video_out_gpu_next,
+    &video_out_gpu,
```

Comment:
Can we keep vo_gpu for Android as first one? This is kinda blocker for gpu-next https://github.com/mpv-player/mpv/issues/13706 on Android.

Reply:
I'm not sure this is necessary because mpv isn't used directly on Android, mpv-android or other similar clients can just set the vo and we can add a note for this in the docs.

Reply:
fundamental question is: should we have defaults where we know some platforms/hardware/content/... has relevant problems working correctly?

effectively we're already doing that because the typical android potato GPU breaks with our (not so new) default scalers, so mpv-android already applies `profile=fast`.

Reply:
Android also breaks with our (not so new) default Vulkan, which is why it is excluded.

Reply:
Ah right, I forgot about hwdec.